tests/Backend/Snowflake/DirectAccessTest.php

```diff
@@ -471,6 +482,88 @@ public function testDirectAccessAndRestrictions()
             . ' AS SELECT * FROM "in.c-API-DA_TEST"."other_table"',
             $views[0]['text']
         );
+
+
+        $this->_client->createAliasTable($bucketId, $tableId, 'this-is-alias');
```

Comment:
v zdrojovom buckete vytvorim alias aby som otestoval ze to vtedy nejde zapnut v tom nalinkovanom buckete